app/src/main/java/jetzt/machbarschaft/android/service/testapi/data/CreateUserBody.kt

```diff
@@ -0,0 +1,134 @@
+package jetzt.machbarschaft.android.service.testapi.data
+
+import com.google.gson.annotations.SerializedName
+
+
+/**
+ * This class was created by hand and needs to be replaced with the automatic created class from OPEN API Code Generator. :)
+ * TODO replace
+ */
+
+class CreateUserBody {
```

Comment:
Constructor, `toString()` and `equals()` can be replaced by making this (and `GeoPoint`) a data class. The class properties have to be moved to the class definition, otherwise they are not included in the constructor, see [here](https://kotlinlang.org/docs/reference/data-classes.html#properties-declared-in-the-class-body).

Reply:
Done